### PR TITLE
New kabaddi Terms Cheat Sheet

### PR DESCRIPTION
**Pull request by sinwar**
What does your Instant Answer do?
It tells about terms used in kabadii and its definitions.

What is the data source for your Instant Answer? (Provide a link if possible)
Wikipedia, https://en.wikipedia.org/wiki/Kabaddi

Why did you choose this data source?
This is perfect.

Are there any other alternative (better) data sources?
Ya, http://www.prokabaddi.com/prokabaddi-rules

Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)
Kabaddi lovers 

Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?
yup

Are you having any problems? Do you need our help with anything?
Nope, thanks

Where did you hear about DuckDuckHack? (For first time contributors)
from hackathon sponsered by DuckDuckGo

What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)
kabaddi

-----
IA Page: https://duck.co/ia/view/kabaddi_terms_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@sinwar Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by sinwar** [comment]
@moollaza checkout this PR?

File: share/goodie/cheat_sheets/json/kabaddi-terms.json [hunk 1]
```diff
@@ -0,0 +1,97 @@
+{
+    "id":"kabaddi_terms",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/kabaddi-terms.json]: please change this to `kabaddi_terms_cheat_sheet`

File: share/goodie/cheat_sheets/json/kabaddi-terms.json [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    "name":"Kabaddi Terms and its definitions",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/kabaddi-terms.json]: Please change to `Kabaddi Terms and Definitions`

File: share/goodie/cheat_sheets/json/kabaddi-terms.json [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    "description": "Kabaddi has some terms which are used during matches.",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/kabaddi-terms.json]: Let's reword this: `Common terms used in Kabaddi and their definitions"

File: share/goodie/cheat_sheets/json/kabaddi-terms.json [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        "sourceUrl": "https://en.wikipedia.org/wiki/Kabaddi"
+    },
+    "aliases": ["kabaddi words", "kabaddi definitions", "words used in kabaddi", "terms used in kabaddi"],
+    "template_type": "reference",
+    "section_order": [
+        "Terms and its definitions-I",
+        "Terms and its definitions-II"
+    ],
+    "sections": {
+        "Terms and its definitions-I": [
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/kabaddi-terms.json]: Let's find a better way to categorize these terms. It seems like some describe the court, some describe player, others are "methods" or "tactics", etc. Let's group them accordingly and use sections that suggest more about the terms

File: share/goodie/cheat_sheets/json/kabaddi-terms.json [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+                "key": "To hold a Raider",
+                "val": "It means if the if the defender or defenders hold the raider without any breach in the rules of play and keep the raider in their court and do not allow him to reach his court until he loses his cant or the Referee/Umpire blows the whistle."
+            },
+            {
+                "key": "Touch",
+                "val": "It means if the raider touches any one or more defender by any part of his body including the defenders clothing, shoes and any other apparel he may be wearing."
+            },
+            {
+                "key": "Struggle",
+                "val": "It means when one of more defenders come into contact with the raider. After touch or struggle, the lobbies are activated and included as part of the court."
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/kabaddi-terms.json]: Let's remove `It means`, it's redundant.

File: share/goodie/cheat_sheets/json/kabaddi-terms.json [hunk 6]
```diff
@@ -0,0 +1,97 @@
+{
+    "id":"kabaddi_terms",
+    "name":"Kabaddi Terms and its definitions",
+    "description": "Kabaddi has some terms which are used during matches.",
+    "metadata": {
+        "sourceName": "Wikipedia",
+        "sourceUrl": "https://en.wikipedia.org/wiki/Kabaddi"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/kabaddi-terms.json]: It seems like these descriptions are your own -- is there no official source for the terminology and their meanings? It seem like the Wiki page doesn't provide any, so it's not the idea place to send the user.

File: share/goodie/cheat_sheets/json/kabaddi-terms.json [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+            }
+         ],
+       "Terms and its definitions-II": [
+            {
+                "key": "To put out a Defender",
+                "val": "The defender being declared out when a raider touches the defender without any breach in rules of play, or if any part of the body of the defender touches any part of the raider, and then the raider touches his court with the cant."
+            },
+            {
+                "key": "To hold a Raider",
+                "val": "It means if the if the defender or defenders hold the raider without any breach in the rules of play and keep the raider in their court and do not allow him to reach his court until he loses his cant or the Referee/Umpire blows the whistle."
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/kabaddi-terms.json]: Let's make these descriptions more succinct:

For example: "When the defender(s) hold the raider, keeping him in their court, until the raider loses his cant or the whistle blows"

**Comment by moollaza** [comment]
@sinwar this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by moollaza** [comment]
@sinwar congrats! It's live: https://duckduckgo.com/?q=Kabaddi+Terms+Cheat+Sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**